spec/03-plugins/17-jwt/01-jwt_parser_spec.lua

```diff
@@ -29,6 +29,53 @@ describe("Plugin: jwt (parser)", function()
         ]], true), token)
       end
     end)
+    it("should properly encode using HS384", function()
+      local token = jwt_parser.encode({
+        name  = "John Doe",
+        admin = true,
+        sub   = "1234567890"
+      }, "secret", 'HS384')
+
+      if helpers.openresty_ver_num < 11123 then
```

Comment:
Since 0.14.0 (now in the master branch), we only support OpenResty 1.13.2.6  and above, so we do not need these branches anymore. Our CI also runs with recent OpenResty version(s) as well: https://github.com/Kong/kong/blob/master/.travis.yml#L28-L29